### DISCUSSION TITLE
Can't Read Text In Music Player

### DISCUSSION THREAD

**Discussion by ItsEpicBeast** [KRTirtho/spotube#1697] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When clicking a song it's hard to read queue and lyrics option. The whole interface when looking at the player menu is not looking good.
![Screenshot_20240620_002116_Spotube](https://github.com/KRTirtho/spotube/assets/52136474/b6366669-55a6-493f-90fa-416bcd6fda06)
![Screenshot_20240620_002309_Spotube](https://github.com/KRTirtho/spotube/assets/52136474/978ccb4d-e908-4bb4-9bf3-9dea6ba89f8b)

### Expected Behavior

I believe that I didn't have this issues a few months back but I don't remember which versions worked.

### Steps to reproduce

1. Open the app
2. Looked for a song from my playlist 
3. Downloaded the song 
4. When opening the song, it shows the music player
5. The colors for the music player make it hard to interact with the interface

### Operating System

Android 

### Spotube version

v3.7.1

### Installation source

F-Droid (Android)

### Additional information

Either the text colors has to be fixed or the background colors

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by KRTirtho** [comment]
Oh, Spotify's image CDN is failing in your region so the app can't get the Album covers probably. You can try re-installing (or, Clear Data/Cache) the app incase it's a caching problem

**Reply by ItsEpicBeast** [reply]
That's is the album cover, you can look up the song and it's a black cover. My main concern is that the colors for the music player make it hard to interact with the interface.
